charts/golang/templates/ingress.yaml

```diff
@@ -36,36 +35,20 @@ spec:
           {{- if .Values.swagger.enabled }}
           - path: /docs
             {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
-            pathType: {{ .Values.ingress.pathType }}
+            pathType: Prefix
             {{- end }}
             backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "swagger" "context" $)  | nindent 14 }}
-          {{- end }}
-    {{- else }}
-    - host: {{ include "golang.ingressHostname" . | quote }}
-      http:
-        paths:
-          - path: {{ .Values.ingress.path }}
+          - path: /swagger.json
             {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
-            pathType: {{ .Values.ingress.pathType }}
-            {{- end }}
-            backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "http" "context" $)  | nindent 14 }}
-          {{- if .Values.swagger.enabled }}
-          - path: /docs
-            {{- if eq "true" (include "common.ingress.supportsPathType" .) }}
-            pathType: {{ .Values.ingress.pathType }}
+            pathType: Exact
             {{- end }}
             backend: {{- include "common.ingress.backend" (dict "serviceName" (include "common.names.fullname" .) "servicePort" "swagger" "context" $)  | nindent 14 }}
           {{- end }}
```

Comment:
We need to define `/docs` and `/swagger.json`. I tried doing a rewrite with `/docs(/|$)(.*)` and the `rewrite-target` header, but that ended up breaking the app requests. We can improve this later if needed, but this got our specific swagger stuff working.